error.go

```diff
@@ -140,11 +141,10 @@ func WrapPrefix(e interface{}, prefix string, skip int) *Error {
 }
 
 // Is detects whether the error is equal to a given error. Errors
-// are considered equal by this function if they are the same object,
-// or if they both contain the same error inside an errors.Error.
+// are considered equal by this function if they are matched by errors.Is
+// or if their contained errors are matched through errors.Is
 func Is(e error, original error) bool {
-
-	if e == original {
+	if baseErrors.Is(e, original) {
```

Comment:
N.B. This will only compile on go1.13 and above – given that go only supports the two most recent major releases, I think that it's ok for us to drop go1.12 support; but we should add something to the README to let people know that this has happened.

Could you please add a section 

```
## Changelog
* v1.1.0 updated to use go1.13's standard-library errors.Is method instead of == in errors.Is
```